rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -3,20 +3,23 @@
 %global foreman_proxy_dir /usr/share/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root/-}%{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
The `/-` after `..../tfm/root` should be removed as it generates this path with a hyphen in: `/opt/theforeman/tfm/root/-/usr/share/smart_proxy_dynflow_core/bundler.d/foreman_remote_execution_core.rb`